### PR TITLE
 Enable non-English markdown instructions behind experiment flag

### PR DESCRIPTION
**Pull request by Hamms**
TODO:

As it currently stands, the markdown instructions going in to crowdin have a bunch of `\r` carriage return characters, and the markdown instructions coming back _out_ of crowdin have a bunch of `\\r` _escaped_ carriage return characters, which messes things up

image

I'd like to see if I can easily get that fixed before merging this

### CODE REVIEW

**Comment by Hamms** [comment]
[Turn on markdown instructions for other languages](https://trello.com/c/z4MfiLY6/101-turn-on-markdown-instructions-for-other-languages)

**Comment by Hamms** [comment]
Well, it wasn't _easy_ per se, but https://github.com/code-dot-org/code-dot-org/pull/22139 fixes the carriage return issue

File: dashboard/app/models/levels/blockly.rb [hunk 1]
```diff
@@ -371,6 +371,13 @@ def localized_instructions
     end
   end
 
+  def localized_markdown_instructions
+    if should_localize? && markdown_instructions
+      I18n.t("data.markdown_instructions").
+        try(:[], "#{name}_markdown_instruction".to_sym)
+    end
+  end
```
Comment (joshlory) [inline dashboard/app/models/levels/blockly.rb:379]: Any way to dedupe this against similar code (extract a helper)?

```ruby
def localized_failure_message_override
  if should_localize? && failure_message_override
    I18n.t("data.failure_message_overrides").
      try(:[], "#{name}_failure_message_override".to_sym)
  end
end
```

Comment (Hamms) [reply dashboard/app/models/levels/blockly.rb:379]: good call; I'll open up a separate PR

**Pull request merged**